apps/dashboard/src/app/(dashboard)/interest-group/[id]/edit-card.tsx

```diff
@@ -33,11 +33,22 @@ export const InterestGroupEditCard: FC = () => {
     defaultValues: interestGroup,
   })
   return (
-    <div>
+    <Stack gap="xs">
       <FormComponent />
-      <Button variant="outline" color="red" style={{ marginTop: "1rem" }} onClick={open}>
-        <Icon icon="tabler:trash" />
-      </Button>
-    </div>
+      <div style={{ display: "flex", flexWrap: "wrap", alignItems: "center" }}>
+        <Button variant="outline" color="red" style={{ width: "55px", marginRight: "24px" }} onClick={open}>
```

Comment:
This is unrelated but why are we using inline styling in the dashboard but tailwind elsewhere??

Reply:
Mantine components have props that work very similarly to tailwind, see https://mantine.dev/styles/style-props/. You should use these instead of the inline styling 👍

Reply:
Fixed